weblate/api/serializers.py

```diff
@@ -168,6 +169,9 @@ class UserSerializer(serializers.ModelSerializer):
     groups = serializers.HyperlinkedIdentityField(
         view_name="api:group-detail", lookup_field="id", many=True, read_only=True,
     )
+    subscriptions = serializers.HyperlinkedIdentityField(
+        view_name="api:user-notifications", lookup_field="username",
+    )
```

Comment:
I'd probably call this notifications as well to be consistent in the API. The reason I don't want to use "subscriptions" here because it's confusing - notification subscription or service subscription.

Reply:
I will fix this in master branch as I've merged this accidentally when pushing other changes....

Reply:
ok.